File: contest/ABC115/py/b.py
n = int(input())
l = list(int(i) for i in input().split())

l_max = max(l)
index_max = l.index(l_max)

sum = sum(l) - l_max

if (sum > l_max):
    print('Yes')
else:
    print('No')

File: contest/ABC115/b.cpp
#include<iostream>

using namespace::std;

int main(int) {
  int N;
  cin >> N;

  int sum = 0, mx = 0;
  for (int i=0; i<N; i++){
    int p;
    cin >> p;
    sum += p;
    mx = max(mx, p);
  }
  cout << sum - (mx / 2);
}